packages/controller/src/resources/app/app.ts

```diff
@@ -48,26 +47,18 @@ export function OpstraceApplicationResources(
     auth0_client_id = "5MoCYfPXPuEzceBLRUr6T6SAklT2GDys";
   }
 
-  collection.add(
```

Comment:
Why are you not creating this namespace?

Reply:
I though[ it's not needed anymore](https://github.com/opstrace/opstrace/pull/749/files#diff-0fb8ba6634d6612e91f37ec5a7104c8911ecb51490391cc8a8b4d4c90aef335eL59-L61), as we [copy the cert now directly](https://github.com/opstrace/opstrace/pull/749/files#diff-0fb8ba6634d6612e91f37ec5a7104c8911ecb51490391cc8a8b4d4c90aef335eR50-R61).

But I realise we probably only need the [commented label](https://github.com/opstrace/opstrace/pull/749/files#diff-0fb8ba6634d6612e91f37ec5a7104c8911ecb51490391cc8a8b4d4c90aef335eL62).